lib/hackmode/realmode.js

```diff
@@ -0,0 +1,1159 @@
+const msgpack = require('msgpack-lite')
```

Comment:
should we move the "real" stuff out of the hackmode directory?

Reply:
My plan is Hackmode is temporary and will be removed in a near future.
RealMode needs all the other stuff in hackmode folder.

Reply:
:+1: